crates/symbolicator-service/src/services/sourcemap_lookup.rs

```diff
@@ -547,9 +547,20 @@ impl ArtifactFetcher {
                     .fetch_scoped_url(&self.scope, url.to_owned())
                     .await;
 
-                return scraped_file.map(|contents| CachedFile {
-                    contents,
-                    sourcemap_url: None,
+                return scraped_file.map(|contents| {
+                    let sm_ref = locate_sourcemap_reference(Cursor::new(&*contents))
```

Comment:
```suggestion
                    let sm_ref = locate_sourcemap_reference(contents.as_bytes())
```

also, this whole change makes sense to land as a separate PR, as it is about following sourcemap references from scraped files.